server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1899,7 +1906,8 @@ public void checkIfMinorCompactionNeededForLogs(List<DfsLogger> closedLogs) {
 
     if (reason != null) {
       // initiate and log outside of tablet lock
-      initiateMinorCompaction(MinorCompactionReason.SYSTEM);
+      TableState tableState = context.getTableManager().getTableState(extent.tableId());
+      initiateMinorCompaction(MinorCompactionReason.SYSTEM, tableState);
       log.debug("Initiating minor compaction for {} because {}", getExtent(), reason);
```

Comment:
It could get confusing that this debug statement says it's initiating a flush and occurs immediately after the one inside `initiateMinorCompaction()` that says not to flush.

Reply:
Good catch. I'll move the debug before the method call.